qiskit/circuit/quantumcircuit.py

```diff
@@ -4776,6 +4776,12 @@ def crx(
         """
         from .library.standard_gates.rx import CRXGate
 
+        # if the control state is |1> use the fast Rust version of the gate
+        if ctrl_state is None or ctrl_state in ["1", 1]:
```

Comment:
Hmm I get that you added the `["1", 1]` cases because it's the state set by default when `ctrl_state` is None, but I wonder if we would be overseeing some edge case by adding this?? (If there is indeed no difference, I think we should do it in all controlled gate methods).

Reply:
If `ctrl_state` is `None` it is directly set to 1 in the constructor of any controlled gate, e.g.
```
>>> CRXGate(theta=0.5, ctrl_state=None).ctrl_state 
1
```
so I think there should be no functional difference 🙂 

But I agree we should generally cover all `|1>` state definitions we can use the Rust implementation more generally.